File: backend/logistics-users-service/src/main/java/com/tdtu/logistics_users_service/dto/response/ReceiverInfResponse.java
package com.tdtu.logistics_users_service.dto.response;

import lombok.Data;

import java.util.UUID;

@Data
public class ReceiverInfResponse {
    UUID id; // ID của người nhận (tự sinh).

    UUID customerId; // ID của khách hàng.

    String fullName; // Tên đầy đủ của người nhận.

    String phoneNumber; // Số điện thoại của người nhận.

    String email; // Email liên hệ.

    String province; // Tỉnh.

    String district; // Huyện.

    String ward; // Xã/Phường.

    String street; // Đường.

    String postalCode; // Mã bưu chính.
}

File: my-app/src/modules/order/models/ReceiverInfResponse.ts
export interface ReceiverInfResponse {
    id: string; // ID của người nhận (tự sinh).
    customerId: string; // ID của khách hàng.
    fullName: string; // Tên đầy đủ của người nhận.
    phoneNumber: string; // Số điện thoại của người nhận.
    email: string; // Email liên hệ.
    province: string; // Tỉnh.
    district: string; // Huyện.
    ward: string; // Xã/Phường.
    street: string; // Đường.
    postalCode: string; // Mã bưu chính.
}
